snc.sh

```diff
@@ -256,12 +256,16 @@ function delete_operator() {
         ${OC} wait --for=delete pod/${pod} --timeout=120s -n ${namespace} || ${OC} delete pod/${pod} --grace-period=0 --force -n ${namespace} || true
 }
 
+# Download the oc binary for all platforms
+mkdir -p openshift-clients/linux openshift-clients/mac openshift-clients/windows
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/linux oc
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/mac oc
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows-${OPENSHIFT_RELEASE_VERSION}.zip" > openshift-clients/windows/oc.zip
+unzip -o -d openshift-clients/windows/ openshift-clients/windows/oc.zip
```

Comment:
I was thinking to use something following to fetch the oc from release artifacts but then the extracted image version is not same as it suppose to be.

```
$ oc adm release info -a ~/pull-secret quay.io/openshift-release-dev/ocp-release@sha256:e740f474eca36f9294969bc806559bc00f9858b3ca828012a1da99d9f0570bfd --image-for=cli-artifacts
$ oc image extract -a ~/pull-secret quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:79978a34d1ab3b0ed1ad2c93c09bcb2fcdd1806b35e48a53c99d106347e1a59d --path=/usr/share/openshift/linux_amd64/oc:.  --confirm

$ ./oc version
Client Version: openshift-clients-4.6.0-202006250705.p0-162-g074039a0a
```